--- jvm-server.options.orig     2025-12-10 14:43:25.853880972 -0800
+++ jvm-server.options  2025-12-10 14:45:06.774444355 -0800
@@ -165,6 +165,9 @@
 #
 # For production use you may wish to adjust this for your environment.
 # If that's the case, see MAX_HEAP_SIZE (and HEAP_NEWSIZE for CMS) in cassandra-env.sh
+-Xms64G
+-Xmx64G
+-XX:-UseCompressedOops

 #####################
 # OFF-HEAP SETTINGS #
@@ -186,3 +189,6 @@
 # * CAP_NOWARN: same as previous, except that the client warning will not be emitted.
 #
 #-Dcassandra.expiration_date_overflow_policy=REJECT
+
+# Allow number of validators to exceed number of compactors.
+-Dcassandra.allow_unlimited_concurrent_validations=true
