--- jvm17-server.options.orig   2025-12-11 14:13:59.209357678 -0800
+++ jvm17-server.options        2025-12-11 14:15:50.255994053 -0800
@@ -51,7 +51,7 @@
 ## Optional G1 Settings
 # Save CPU time on large (>= 16GB) heaps by delaying region scanning
 # until the heap is 70% full. The default in Hotspot 8u40 is 40%.
--XX:InitiatingHeapOccupancyPercent=70
+-XX:InitiatingHeapOccupancyPercent=40

 # For systems with > 8 cores, the default ParallelGCThreads is 5/8 the number of logical cores.
 # Otherwise equal to the number of cores when 8 or less.
@@ -59,8 +59,8 @@
 # By default, ConcGCThreads is 1/4 of ParallelGCThreads.
 # Setting both to the same value can reduce STW durations.
 # When leaving both unset then cassandra-env.sh will set them both to the number of your cores.
-#-XX:ParallelGCThreads=16
-#-XX:ConcGCThreads=16
+-XX:ParallelGCThreads=32
+-XX:ConcGCThreads=32


 ### JPMS
@@ -95,7 +95,7 @@
 # Java 11 (and newer) GC logging options:
 # See description of https://bugs.openjdk.java.net/browse/JDK-8046148 for details about the syntax
 # The following is the equivalent to -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=10M
-#-Xlog:gc=info,heap*=trace,age*=debug,safepoint=info,promotion*=trace:file=/var/log/cassandra/gc.log:time,uptime,pid,tid,level:filecount=10,filesize=10485760
+-Xlog:gc=info,heap*=info,age*=debug,safepoint=info,promotion*=trace:file=/var/log/cassandra/gc.log:time,uptime,pid,tid,level,tags:filecount=10,filesize=10485760

 # Notes for Java 8 migration:
 #
